File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        set <int> check;
        for (auto n : nums) {
            if (check.contains(n)) {return 1;}
            else {check.insert(n);}
        }
        return 0;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet <Integer> check = new HashSet<Integer>();

        for (var n : nums) {
            if (check.contains(n)) {return true;}
            else {check.add(n);}
        }
        return false;
    }
}